lib/galaxy/tool_util/deps/container_resolvers/mulled.py

```diff
@@ -407,17 +407,20 @@ def resolve(self, enabled_container_types, tool_info, install=False, session=Non
                 type=self.container_type,
                 shell=self.shell,
             )
-            destination_for_container_type = kwds.get('destination_for_container_type')
-            if install and destination_for_container_type and not self.cached_container_description(
```

Comment:
It's this bit right?  It dropped the bypass when a destination is trusted to understand a container type implicitly?

Reply:
Are you looking at a bug or weird behavior ? If you wanted to fetch a container previously you had to pass in `destination_for_container_type `, which you don't have available when not preparing a job, for instance when pulling an image from the admin panel. Is this causing excessive pulls, and if so, what does the container resolution config look like ?

Reply:
Ah, just saw the issue now.